src/mixins/createFormFieldMixin.ts

```diff
@@ -17,14 +18,18 @@ export interface FormFieldMixinOptions<V, S extends FormFieldMixinState<V>> exte
 	 * The value of the form field
 	 */
 	value?: V;
+
 }
 
-export interface FormFieldMixinState<V> {
-	/**
-	 * Whether the field is currently disabled or not
-	 */
-	disabled?: boolean;
+export interface FormFieldMixinProperties extends WidgetProperties {
+	label?: string | {
```

Comment:
Content could be mandatory, that seems like a good idea. Did you want `hidden` to be a string instead of a boolean, or was that just a mistype?